lib/audio/sounds.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:audioplayers/audioplayers.dart';
+
+class Sounds {
+  static AssetSource buttonTap = AssetSource('audio/button_tap.mp3');
+  static AssetSource artifactCrafted = AssetSource('audio/artifact_crafted.mp3');
+  static AssetSource toggleSound = AssetSource('audio/toggle_sound.mp3');
+}
```

Comment:
The same, all we need is to regenerate assets

Reply:
Done